apache/flagon-distill
# chore(testing): Installed tox for python versions 3.8 to 3.11

This ticket adds `tox` for testing our code across different python versions. Note python 3.12 is not currently included in the testing as it will cause dependency issues, though if needed we can add it and attempt to resolve them. 

# To test:
Install pyenv: https://github.com/pyenv/pyenv?tab=readme-ov-file#installation
Note: *make sure the .pyenv directory is added to your PATH*
Install tox: 
```sh 
python -m pip install --user tox
```

Run:
```sh
make install_py_versions
make test_all
```

Closes #52
